keck_vnc_launcher.py

```diff
@@ -1093,9 +1094,13 @@ def calc_window_geometry(self):
 
         #get screen dimensions
         #alternate command: xrandr |grep \* | awk '{print $1}'
+        self.geometry = list()
         cmd = "xdpyinfo | grep dimensions | awk '{print $2}' | awk -Fx '{print $1, $2}'"
         p1 = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
         out = p1.communicate()[0].decode('utf-8')
+        if not out:
```

Comment:
I recommend `if out is None:` here for consistency and to make it very explicit.